.github/workflows/super_diff.yml

```diff
@@ -36,3 +36,32 @@ jobs:
         run: bin/ci/install
       - name: Run tests
         run: bin/ci/run-tests
+  test_ruby_3:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby: 3.0
```

Comment:
GitHub Actions doesn't seem to like this (see the Checks tab). Maybe "3.0" should be a string?